scripts/malloc-free.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+# uftrace-option: --force --nest-libcall -A malloc@arg1 -R malloc@retval -A free@arg1 -A calloc@arg1,arg2 -R calloc@retval -A realloc@arg1,arg2 -R realloc@retval -A memalign@arg1,arg2 -R memalign@retval -A malloc@libstdc++,arg1 -R malloc@libstdc++,retval -A free@libstdc++,arg1 -A calloc@libstdc++,arg1,arg2 -R calloc@libstdc++,retval -A realloc@libstdc++,arg1,arg2 -R realloc@libstdc++,retval -A memalign@libstdc++,arg1,arg2 -R memalign@libstdc++,retval
```

Comment:
I think it can be simply `--nest-libcall --auto-args` as you set `UFTRACE_FUNCS` below.